pyppeteer/page.py

```diff
@@ -1013,6 +1013,14 @@ async def setBypassCSP(self, enabled: bool) -> None:
         await self._client.send('Page.setBypassCSP', {'enabled': enabled})
 
     async def emulateMedia(self, mediaType: str = None) -> None:
+        """Deprecated in favor of ``emulateMediaType()``.
+        Please, consider to use ``emulateMediaType()`` instead.
+        """
```

Comment:
```suggestion
        """Deprecated alias for ``emulateMediaType()``"""
```
I think this is a more concise message.